packages/core/echo/echo-schema/src/text-object.ts

```diff
@@ -71,14 +72,13 @@ export class Text extends EchoObject<TextModel> {
   }
 
   protected override _afterBind() {
-    // log.info('_afterBind', { id: this.id });
+    log('_afterBind', { id: this.id });
     this._model.initialize();
   }
 
   override _itemUpdate(): void {
-    // log.info('_itemUpdate', { id: this.id });
-    // TODO(wittjosiah): This stops yjs updates from working.
-    // super._itemUpdate();
+    log('_itemUpdate', { id: this.id });
+    super._itemUpdate();
```

Comment:
Uncommenting this didn't seem to reintroduce the issues we were seeing before :shrug: